node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function cp() {
+  const filePath = new URL(process.argv[2], import.meta.url);
+  const contents = await readFile(filePath, { encoding: 'utf8' });
```

Comment:
By passing 'utf8' as the encoding you limit this code to only working with text files. If you leave off the encoding entirely it will also work with binary files.

Reply:
updated